app/main.py

```diff
@@ -1,52 +1,44 @@
+from app.serialize import JSONSerializeStrategy, XMLSerializeStrategy
 
 
-class Book:
-    def __init__(self, title: str, content: str):
-        self.title = title
-        self.content = content
-
-    def display(self, display_type: str) -> None:
-        if display_type == "console":
-            print(self.content)
-        elif display_type == "reverse":
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown display type: {display_type}")
-
-    def print_book(self, print_type: str) -> None:
-        if print_type == "console":
-            print(f"Printing the book: {self.title}...")
-            print(self.content)
-        elif print_type == "reverse":
-            print(f"Printing the book in reverse: {self.title}...")
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown print type: {print_type}")
+def main(book: Book, commands: list[tuple[str, str]]) -> None | str:
+    display_strategies = {
+        "console": ConsoleDisplayStrategy(),
+        "reverse": ReverseDisplayStrategy(),
+    }
 
-    def serialize(self, serialize_type: str) -> str:
-        if serialize_type == "json":
-            return json.dumps({"title": self.title, "content": self.content})
-        elif serialize_type == "xml":
-            root = ET.Element("book")
-            title = ET.SubElement(root, "title")
-            title.text = self.title
-            content = ET.SubElement(root, "content")
-            content.text = self.content
-            return ET.tostring(root, encoding="unicode")
-        else:
-            raise ValueError(f"Unknown serialize type: {serialize_type}")
+    print_strategies = {
+        "console": ConsolePrintStrategy(),
+        "reverse": ReversePrintStrategy(),
+    }
 
+    serialize_strategies = {
+        "json": JSONSerializeStrategy(),
+        "xml": XMLSerializeStrategy(),
+    }
 
-def main(book: Book, commands: list[tuple[str, str]]) -> None | str:
     for cmd, method_type in commands:
         if cmd == "display":
-            book.display(method_type)
+            strategy = display_strategies.get(method_type)
+            if strategy:
+                strategy.display(book.content)
+            else:
+                raise ValueError(f"Unknown display type: {method_type}")
         elif cmd == "print":
-            book.print_book(method_type)
+            strategy = print_strategies.get(method_type)
+            if strategy:
+                strategy.print_book(book.title, book.content)
+            else:
+                raise ValueError(f"Unknown print type: {method_type}")
         elif cmd == "serialize":
-            return book.serialize(method_type)
+            strategy = serialize_strategies.get(method_type)
+            if strategy:
+                return strategy.serialize(book.title, book.content)
+            else:
+                raise ValueError(f"Unknown serialize type: {method_type}")
```

Comment:
```suggestion
                return strategy.serialize(book.title, book.content)
            raise ValueError(f"Unknown serialize type: {method_type}")
```
`else` redundant here